source/features/restore-file.tsx

```diff
@@ -79,32 +86,46 @@ async function handleRestoreFileClick(event: delegate.Event<MouseEvent, HTMLButt
 
 	filesRestored.add(menuItem);
 
-	menuItem.textContent = 'Restoring…';
 	event.preventDefault();
 	event.stopPropagation();
```

Comment:
These should now be dropped because they're keeping the menu open

Reply:
Alright, will remove both L#89 & L#90.